fastlane/metadata/trade_representative_contact_information/country.txt

```diff
@@ -1 +0,0 @@
-Venezuela
```

Comment:
Are you using this information to fill the store? you need change it with Flyve information